plugins/opsgenie/icon_opsgenie/util/api.py

```diff
@@ -101,16 +101,20 @@ def _call_api(self, method: str, url: str, json_data: dict = None, params: dict
             raise PluginException(preset=PluginException.Preset.UNKNOWN, data=response.text)
 
     def _retry_request(self, request_id: str, max_tries: int) -> str:
-        request_response = self._get_request_status(request_id)
-        attempts_counter, delay = 0, 0
-        while not request_response.get("data").get("alertId", "") and attempts_counter < max_tries:
+        request_response = {"data": {}}
+        attempts_counter, delay = -1, 0
+        while not request_response.get("data", {}).get("alertId", "") and attempts_counter < max_tries:
             time.sleep(delay)
-            request_response = self._get_request_status(request_id)
-            attempts_counter += 1
-            delay = 2 ** (attempts_counter * 0.6)
-            self.logger.info(
-                GET_REQUEST_ID_RETRY_MESSAGE.format(delay=delay, attempts_counter=attempts_counter, max_tries=max_tries)
-            )
+            try:
+                request_response = self._get_request_status(request_id)
+            except PluginException:
+                attempts_counter += 1 - 84
```

Comment:
Why you are subtracting `84`?
```suggestion
                attempts_counter += 1
```